vms/platformvm/dac/camino_fee_distribution_proposal.go

```diff
@@ -0,0 +1,233 @@
+// Copyright (C) 2023, Chain4Travel AG. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package dac
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/math"
+	"github.com/ava-labs/avalanchego/utils/set"
+	as "github.com/ava-labs/avalanchego/vms/platformvm/addrstate"
+	"golang.org/x/exp/slices"
+)
+
+const feeDistributionProposalMaxOptionsCount = 3
+
+var (
+	_ Proposal      = (*FeeDistributionProposal)(nil)
+	_ ProposalState = (*FeeDistributionProposalState)(nil)
+
+	InitialFeeDistribution = [3]uint64{30, 30, 40} // 30% validators, 30% grant program, 40% burned
```

Comment:
isn't that maybe some genesis related value that should be mentioned there?

Reply:
It just mirrors value from caminoethvm
But yes, good idea to put this into PR description